src/test/sighash_tests.cpp

```diff
@@ -124,12 +124,12 @@ void static RandomTransaction(CMutableTransaction &tx, bool fSingle) {
             } else {
                 pourtx.vpub_new = insecure_rand() % 100000000;
             }
-            RandomScript(pourtx.scriptPubKey);
-            RandomScript(pourtx.scriptSig);
+
             pourtx.anchor = GetRandHash();
             pourtx.serials[0] = GetRandHash();
             pourtx.serials[1] = GetRandHash();
             pourtx.ephemeralKey = GetRandHash();
+            pourtx.randomSeed = GetRandHash();
             pourtx.ciphertexts[0] = {insecure_rand() % 100, insecure_rand() % 100};
             pourtx.ciphertexts[1] = {insecure_rand() % 100, insecure_rand() % 100};
```

Comment:
Is there a reason this is only setting the first two bytes (or whatever they are) of `ciphertexts[0]` and `ciphertexts[1]`?